### PR TITLE
Add the introduction to LessCode.EFCore.StronglyTypedId

### PR DESCRIPTION
**Pull request by yangzhongke**
Add the introduction to LessCode.EFCore.StronglyTypedId

As proposed on [https://github.com/dotnet/EntityFramework.Docs/issues/4160#issuecomment-1327216363](https://github.com/dotnet/EntityFramework.Docs/issues/4160#issuecomment-1327216363)

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -342,6 +342,14 @@ For EF Core: 6, 7.
 
 [GitHub repository](https://github.com/R4ND3LL/EntityFrameworkRuler/) | [CLI Tool NuGet](https://www.nuget.org/packages/EntityFrameworkRuler/) | [Design NuGet](https://www.nuget.org/packages/EntityFrameworkRuler.Design/)
 
+### LessCode.EFCore.StronglyTypedId
+
+A source generator that can generate strongly-typed-id classes automatically for entities.
+
+For EF Core: 7.
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:349]: Move up to a single line

**Comment by roji** [comment]
@yangzhongke note that you don't need to open a new PR (I'm referring to #4163), you can just push changes to an existing PR's branch.

**Comment by yangzhongke** [comment]
> @yangzhongke note that you don't need to open a new PR (I'm referring to #4163), you can just push changes to an existing PR's branch.

Thanks. I'm sorry that I have already deleted the previous branch. 
I have 'Moved up to a single line'.

**Comment by ErikEJ** [comment]
LGTM

**Review by roji — APPROVED** [review]

**Pull request merged**